core/variant/typed_array.h

```diff
@@ -77,6 +77,9 @@ struct VariantInternalAccessor<const TypedArray<T> &> {
 			ERR_FAIL_COND_MSG(!is_same_typed(p_array), "Cannot assign an array with a different element type."); \
 			_ref(p_array);                                                                                       \
 		}                                                                                                        \
+		_FORCE_INLINE_ TypedArray(std::initializer_list<Variant> p_init) :                                       \
+				Array(Array(p_init), m_variant_type, StringName(), Variant()) {                                  \
```

Comment:
Are you sure this is typesafe? I think this will allow instantiation like:
```c++
TypedArray<int> array = { "string" };
```
Because the array `initializer_list` constructor doesn't do type checks.